juju/client/jujudata.py

```diff
@@ -127,3 +129,11 @@ def _load_yaml(self, filename, key):
             data = yaml.safe_load(f)
             self._loaded[filename] = data
             return data.get(key)
+
+    def cookies_for_controller(self, controller_name):
+        f = pathlib.Path(self.path) / 'cookies' / controller_name
```

Comment:
Ah, this is the first time I see the overloading of / to create a path. Not sure how I feel about that.

Reply:
pathlib is part of the standard library since 3.4.  Personally, I find it much easier to read and understand than `os.path.join(foo, bar, qux, ...)`, and the other methods that the OO syntax allows makes it a clear win to me.